src/Umbraco.Web.UI.Client/src/less/panel.less

```diff
@@ -115,65 +136,76 @@
 }
 
 .umb-panel-footer {
-  margin: 0;
-  padding: 20px;
-  z-index: 999;
-  position: absolute;
-  bottom: 0px;
-  left: 0px;
-  right: 0px;
+    margin: 0;
+    padding: 20px;
+    z-index: 999;
+    position: absolute;
+    bottom: 0px;
+    left: 0px;
+    right: 0px;
 }
 
 
 /* Publish */
 .umb-btn-toolbar .dropdown-menu {
-  right: 0;
-  left: auto;
-  border-radius: @tabsBorderRadius;
-  box-shadow: none;
-  padding: 0
- }
-
- .umb-btn-toolbar .dropdown-menu small {
-  background: #fef9db;
-  display: block;
-  padding: 10px 20px;
- }
-
- .umb-btn-toolbar .dropdown-menu .btn {
+    right: 0;
+    left: auto;
+    border-radius: @tabsBorderRadius;
+    box-shadow: none;
+    padding: 0;
+}
+
+.umb-btn-toolbar .dropdown-menu small {
+    background: #fef9db;
+    display: block;
+    padding: 10px 20px;
+}
+
+.umb-btn-toolbar .dropdown-menu .btn  {
     margin: 20px 29px;
-    width: 80px
- }
+    width: 80px;
+}
 
 /* tab buttons */
-.umb-bottom-bar{
+.umb-bottom-bar {
     background: white;
     -webkit-box-shadow: 0px -18px 20px rgba(255, 255, 255, 1);
-    -moz-box-shadow:    0px -18px 20px rgba(255, 255, 255, 1);
-    box-shadow:         0px -18px 20px rgba(255, 255, 255, 1);
+    -moz-box-shadow: 0px -18px 20px rgba(255, 255, 255, 1);
+    box-shadow: 0px -18px 20px rgba(255, 255, 255, 1);
     border-top: 1px solid @grayLighter;
     padding: 10px 0 10px 0;
     position: fixed;
-    bottom: 0px;
+    bottom: 0;
     left: 100px;
+    right: 40px;
     z-index: 6010;
-}
 
-.umb-tab-buttons{
-    padding-left: 0px;
+    @media (min-width: 1101px) {
+        left: 460px;
+    }
 }
 
-@media (min-width: 1101px) {
-    .umb-bottom-bar {
-        left: 460px;
-    }    
+.umb-tab-buttons {
+    padding-left: 0;
+
+    > .btn-group:not([style*="display:none"]):not([style*="display: none"]) {
+        margin-left: 0;
+    }

```

Comment:
This is because of the "return to list" button is hidden using ```style="display:none;"``` but it could contains a space or not, so both cases are handled. So it just don't use ```margin-left: 5px;``` for the first visible button.